bc-rest/src/main/webapp/WEB-INF/web.xml

```diff
@@ -7,11 +7,11 @@
         <servlet-class>io.swagger.jaxrs.config.DefaultJaxrsConfig</servlet-class>
         <init-param>
             <param-name>api.version</param-name>
-            <param-value>0.3.0</param-value>
+            <param-value>${version.bc.rest}.0</param-value>
```

Comment:
Do we have to set the version as <>.<>.<>, or can we just use <>.<>  ? I have a feeling the trailing '.0' might bite us  sometimes, but I don't remember anymore what does 'api.version' do.